core/store/migrate/migrations/0065_update_check_job_proposals_status_fsm.sql

```diff
@@ -0,0 +1,30 @@
+-- +goose Up
+-- +goose StatementBegin
+
+ALTER TABLE job_proposals
+DROP CONSTRAINT chk_job_proposals_status_fsm;
+
+ALTER TABLE job_proposals
+ADD CONSTRAINT chk_job_proposals_status_fsm CHECK (
```

Comment:
This has to be done over 2 migrations because altering the type and updating the constraint must be done in separate transactions